noir/src/filesystem.rs

```diff
@@ -1,10 +1,38 @@
-use crate::payload::UploadUrls;
 use anyhow::Result;
 use relayer_utils::LOG;
 use sdk_utils::{run_command, upload_to_url};
 use slog::info;
 use std::{fs, path::Path};
 
+use crate::handlers::UploadUrls;
+
+#[cfg_attr(test, mockall::automock)]
+pub trait FileUploader {
```

Comment:
This be added to sdk utils, and UploadUrls can also be some trait since that would be different for circom and noir